conf/script/prov-m0-reset

```diff
@@ -336,6 +336,27 @@ cleanup() {
     run_cmd $node2 "mountpoint -q /var/motr2 && sudo umount /var/motr2 || true"
 }
 
+has_m0d_started() {
```

Comment:
On line # 108 of this file for the case single-node the shift should not have '2' after it.
Please put this fix in cortx-1.0 too.